blog/2022-11-07-rds-privatelink.mdx

```diff
@@ -0,0 +1,77 @@
+---
+slug: rds-privatelink
+title: Securely connect Amazon RDS using PrivateLink
+authors: [depombo, dfellis, aguillenv, yrobla]
+---
+
+# Securely connect Amazon RDS using PrivateLink
+
+<p>
+  Do you have some database instances on RDS and wonder what's the most secure
+  way to reach them?
+</p>
+<p>
+  While RDS endpoints can be reached from public internet, you should consider
+  accessing them internally from your VPC, to grant additional security to those
+  services that are critical.
+</p>
+<p>
+  AWS PrivateLink offers the possibility to securely access those services
+  without exposing them to internet, just using Amazon's private network.
+</p>
+
+## Do you want to know if you have it properly configured? Please use this query to check it:
+
+<!--- https://www.urlencoder.org/ -->
+
+<p>
+  <a href="https://app.iasql.com/#/button/Check RDSA endpoints/----%20Installing%20the%20needed%20modules%0ASELECT%20iasql_install%28%27aws_rds%27%29%3B%0ASELECT%20iasql_install%28%27aws_vpc%27%29%3B%0A%0A--%20Sync%20the%20existing%20data%0Aselect%20%2A%20from%20iasql_sync%28%29%3B%0A--%20Perform%20the%20query%20for%20endpoints%0Aselect%20RDS.region%2C%20vpc.is_default%2C%20vpc.cidr_block%2C%0A%28select%20count%28%2A%29%3E0%20from%20endpoint_interface%20where%20endpoint_interface.region%3DRDS.region%20and%20service%3D%27rds%27%20and%20endpoint_interface.vpc_id%3Dvpc.id%29%20as%20has_endpoint_interface%0Afrom%20RDS%20left%20outer%20join%20vpc%20on%20vpc.region%3DRDS.region%3B">
+    <button class="button button--lg button--primary">
+      Check your endpoints
+    </button>
+  </a>
+</p>
```

Comment:
1) I am not sure TheButton title would work with spaces and no uri escaping? let's rename to `check_RDSA_endpoints` avoid escaping issues
2) it might be worth adding the query in a snippet as well
```sql
SELECT...
```
and then have the button say `Run SQL to check endpoints`